src/Agent.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -223,9 +221,7 @@ public AgentSettings LoadSettings()
                 agentName = consoleWizard.ReadValue(CliArgs.Agent,
                                                 StringUtil.Loc("AgentName"),
                                                 false,
-                                                // TODO: coreCLR doesn't expose till very recently (Jan 15)
-                                                // Environment.MachineName,
-                                                "myagent",
+                                                Environment.MachineName,
```

Comment:
Very unlikely, in windows the behavior is as is ie, netbios name of the machine and in linux its calling gethostname which is basic utility, without this gethostname probably most of the service/apps in linux wont even comeup. Its been there since ages.

https://github.com/dotnet/corefx/issues/4306

Even if its null, in the config manager we dont take empty envionment as input..So this should be ok here..